fastparquet/converted_types.py

```diff
@@ -108,9 +108,14 @@ def convert(data, se):
         else:  # byte-string
             # NB: general but slow method
             # could optimize when data.dtype.itemsize <= 8
-            # NB: `from_bytes` may be py>=3.4 only
-            return np.array([int.from_bytes(d, byteorder='big', signed=True) *
-                             scale_factor for d in data])
+            if PY2:
+                def from_bytes(d):
+                    return int(codecs.encode(d, 'hex'), 16) if len(d) else 0
```

Comment:
It seems `binascii.b2a_hex(d)` is faster than `codecs.encode(d, 'hex')`. I'd also add it's more well-known :-)